src/components/PostInfo/PostInfo.scss

```diff
@@ -1 +1,16 @@
-//
+.PostInfo {
+  margin: 10px auto;
+  width: 90%;
+  border: 1px solid #000;
+  border-radius: 8px;
+  background-color: lightgray;
+  padding: 1em;
+
+  &__title {
+    margin: 0;
+  }
+
+  &__header {
+    margin-bottom: 1em;
+  }
+}//
```

Comment:
:harold:
```suggestion
}
```

Reply:
I struggle with it!) but I always forget about the comments